dash/dash.py

```diff
@@ -979,8 +985,49 @@ def get_asset_url(self, path):
 
         return asset
 
+    def enable_dev_tools(self,
+                         debug=False,
+                         dev_tools_serve_dev_bundles=None):
+        """
+        Activate the dev tools, called by `run_server`. If your application is
+        served by wsgi and you want to activate the dev tools, you can call
+        this method out of `__main__`.
+
+        :param debug: If false no tools will be activated.
```

Comment:
Should we add `If True, then all of the other dev tool settings will be True unless specifically overridden.`